css/index.css

```diff
@@ -112,6 +152,46 @@ p {
     justify-content: center;
 }
 
+.boxes .box:nth-child(1) {
+    background: teal;
+}
+
+.boxes .box:nth-child(2) {
```

Comment:
Awesome, you used the "nth-child" selector to dynamically change each color on the box. Another way you could achieve the same result is by keeping the "box" class and add an additional class to each box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```